atom/index.d.ts

```diff
@@ -1,12 +1,4 @@
-export const STORE_CLEAN_DELAY: number
-
-type ReadonlyIfObject<Value> = Value extends undefined
-  ? Value
-  : Value extends (...args: any) => any
-  ? Value
-  : Value extends object
-  ? Readonly<Value>
-  : Value
+type ReadonlyIfObject<Value> = Value extends object ? Readonly<Value> : Value
```

Comment:
```ts
type B = ReadonlyIfObject<(a: number) => number>
```

`B` will be `{}` type. We need this check for `function`

Reply:
https://www.typescriptlang.org/play?#code/C4TwDgpgBAglC8UAUBDAXFAdgVwLYCMIAnASgQD4s9CiAoW0SKAJQhQBMB7TAGxAEkAZgHl8AKwgBjYAB4Aaih7YIlRAqXQIAD2ARM7AM5RO4qcCgB+Fmy68Q8xcsoZ1y+o2gAhBNY7c+QqIS0jKoGDgExGTwlBE05EA